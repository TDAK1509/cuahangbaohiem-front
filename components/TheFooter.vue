<template>
  <footer class="footer has-background-dark has-text-white">
    <div class="columns">
      <div class="column">
        <figure class="image">
          <img class="image footer__image" src="~assets/images/logo.png" />
        </figure>

        <ul class="footer__list is-size-7 ml-4 mt-4">
          <li class="mt-1">Công ty cổ phần Cửa Hàng Bảo Hiểm</li>
          <li class="mt-1">
            Số 69, đường Lê Lai, phường Lê Lợi,<br />Thành phố Hồ Chí Minh
          </li>

          <li class="mt-1">
            Hotline: <a class="footer__link" href="tel:12345">12345</a>
          </li>
          <li class="mt-1">
            Email:
            <a class="footer__link" href="mailto:info@cuahangbaohiem.com"
              >info@cuahangbaohiem.com</a
            >
          </li>
        </ul>
      </div>
      <div class="column">
        <h4 class="is-size-5">COLUMN 2</h4>
      </div>
      <div class="column">
        <h4 class="is-size-5">COLUMN 3</h4>
      </div>
    </div>
  </footer>
</template>

<script>
export default {
  name: "Footer"
};
</script>

<style lang="scss" scoped>
.footer__image {
  height: 40px;
  width: auto;
}

.footer__list {
  list-style-type: disc;
}

.footer__link {
  color: yellow;

  &:hover {
    color: #fff;
    text-decoration: underline;
  }
}
</style>
