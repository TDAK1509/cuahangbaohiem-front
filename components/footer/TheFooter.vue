<template>
  <footer class="the-footer footer has-background-dark has-text-white">
    <div class="columns">
      <FooterFirstColumn />
      <FooterSecondColumn />
      <FooterThirdColumn />
    </div>
  </footer>
</template>

<script>
export default {
  name: "Footer"
};
</script>

<style lang="scss" scoped>
.the-footer {
  margin-top: auto;
}
</style>
