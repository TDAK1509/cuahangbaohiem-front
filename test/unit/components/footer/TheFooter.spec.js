import { mount } from "@vue/test-utils";
import TheFooter from "@/components/footer/TheFooter.vue";

describe("TheFooter.vue", () => {
  it("matches snapshot", () => {
    const wrapper = mount(TheFooter);
    expect(wrapper.element).toMatchSnapshot();
  });
});
