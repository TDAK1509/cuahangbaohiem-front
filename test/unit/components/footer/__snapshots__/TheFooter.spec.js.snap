// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheFooter.vue matches snapshot 1`] = `
<footer
  class="the-footer footer has-background-dark has-text-white"
>
  <div
    class="columns"
  >
    <footerfirstcolumn />
     
    <footersecondcolumn />
     
    <footerthirdcolumn />
  </div>
</footer>
`;
